src/controllers/trainee/Controller.ts

```diff
@@ -0,0 +1,112 @@
+import { Request, Response } from 'express';
+import { UserRepository } from '../../repositories';
+
+const userRepository = new UserRepository();
+
+class TraineeRoutes {
+// Function to fetch all trainees
+    public async get(request: Request, response: Response, next) {
+        try {
+            const {skip , limit} = request.query;
```

Comment:
Why are we not using interface over here?
const {skip , limit} = request.query as IQueryTrainee